stable/storages/templates/NOTES.txt

```diff
@@ -0,0 +1 @@
+storages chart is used with conjunction of other charts. It can come in handy when a public chart does not provide support for additional pv or pvc. In that scenario, this chart can be used to deploy additional pv and pvc.
```

Comment:
NOTES.txt is intended for post installation messages. This is a restrment of purpose.

Reply:
Updated